internal/cadence/cadence.go

```diff
@@ -44,4 +44,5 @@ var Cmd = &cobra.Command{
 
 func init() {
 	Cmd.AddCommand(languageserver.Cmd)
+	lintCommand.AddToParent(Cmd)
```

Comment:
is this the correct place to put it? Seems odd that its the only one in this file

Reply:
Yup, this is the same as every other command.  The difference between this command and the `languageserver.Cmd` is that the latter isn't actually using the `command.Command` wrapper that we use for other commands so it looks different